codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandlerBuilder.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.codec.http2;
+
+import io.netty.handler.codec.http2.Http2HeadersEncoder.SensitivityDetector;
+
+/**
+ * Builder which builds {@link Http2ConnectionHandler} objects.
+ */
+public final class Http2ConnectionHandlerBuilder
+        extends AbstractHttp2ConnectionHandlerBuilder<Http2ConnectionHandler, Http2ConnectionHandlerBuilder> {
+
+    @Override
+    public Http2ConnectionHandlerBuilder validateHeaders(boolean validateHeaders) {
```

Comment:
I wouldn't say I need total control but I need to enforce the value of certain properties. In such a case, it's always better making the setter method invisible than making it throw an exception.